### ISSUE TITLE
ff-kebab-menu missing features 

### ISSUE THREAD

**Issue by Steve-Mcl** [flowforge/forge-ui-components#76]
### Current Behavior

It is the intention of removing the headlessui MenuItem (as currently used in the action menu) but in its current form lacks several features e.g. disabling menuitems, keyboard navigation, role and aria attributes, separators etc.

### Expected Behavior

as a matter of course, we should implement keyboard navigation and aria / role for accessibility on the menu like the headlessui menu does,

each item in the menu should get a suitable role for screenreaders

As for disabling an item, this could of course be "hacked" (wrapped in a div for example) but supporting the disabled property in the `ff-list-item` might be more consistent (site wide)

 For separators, again this could be manufactured with HTML but feel the kebab menu should have an awareness / support separators O.O.T.B 

### Steps To Reproduce

See more info in https://github.com/flowforge/flowforge/pull/1114

### Environment

- FlowForge version: 0.10.0-git

**Comment by joepavitt** [comment]
update to this in that we do now support `disabled` on `ff-list-item` in a `ff-kebab-menu`, see example: https://flowforge.github.io/forge-ui-components/#ff-kebab-menu
